src/main/java/com/Autopark/infrastructure/orm/service/PostgreDataBaseService.java

```diff
@@ -132,6 +157,26 @@ public <T> Optional<T> get(Long id, Class<T> clazz) {
         return Optional.empty();
     }
 
+    @SneakyThrows
+    public <T> Optional<T> getColumn(Long id, Class<T> clazz) {
+        checkTableAnnotation(clazz);
+
+        String sql = "SELECT * FROM " + clazz.getDeclaredAnnotation(Table.class).name() +
```

Comment:
separate to the variable of the class

How it's did for another SQL requests

Reply:
fixed